### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#216] [Esercizi - Bertini] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8d2d0800-df12-4b41-91c2-85e157f9dd5f)

**Comment by cornflexxx** [comment, marked as answer]
## Soluzione analoga a quella in classe (problema generalizzato)
$1)$ Dobbiamo calcolare la distribuzione della somma di tutte le $Z_i$, possiamo partire calcolando la distribuzione somma di due $v.a$ aleatorie con dist. di Poisson. A partire da quella somma viene automatico calcolare la somma di $k$ termini.

Definiamo $X_1,X_2$ , entrambe con distribuzione di Poisson con parametri $\lambda_1,\lambda_2$, e definiamo poi la loro somma $X=X_1+X_2$, diciamo che $X=x$, allora la sua distribuzione si calcola così:

$$\mathbb{P}(X=x)=\mathbb{P}(X_1=x_1,X_2=x-x_1)=\sum_{i=0}^x\mathbb{P}(X_1=i,X_2=x-i)=\sum_{i}^x e^{-\lambda_1} \frac{\lambda_1^i}{i!}\cdot e^{-\lambda_2}\frac{\lambda_2^{x-i}}{(x-i)!}=...$$

$$...=e^{-(\lambda_1+\lambda_2)}\sum_{i=0}^x \frac{1}{(x-i)!i!}\lambda_1^i\lambda_2^{x-i}=\frac{1}{x!} e^{-(\lambda_1+\lambda_2)}\sum_{i=0}^x \frac{x!}{(x-i)!i!}\lambda_1^i\lambda_2^{x-i}=...$$

$$...=\frac{1}{x!} e^{-(\lambda_1+\lambda_2)}\sum_{i=0}^x \binom{x}{i}\lambda_1^i\lambda_2^{x-i}=\frac{1}{x!} e^{-(\lambda_1+\lambda_2)}\cdot (\lambda_1+\lambda_2)^x$$


$$X_1+X_2\sim P(\lambda_1+\lambda_2)$$

Generalizzando a $k$ somme  allora otteniamo $P(\lambda_1+...+\lambda_k)$. 

$2)$  Calcolo la congiunta delle $Z_i$ , condiziona al fatto che la loro somma sia $n$:


$$\mathbb{P}(Z_1=n_1,...,Z_k=n_k|Z_1+...+Z_k=n)=\frac{\mathbb{P}(Z_1=n_1,...,Z_k=n_k)}{\mathbb{P}(Z_1,...,Z_k=n)}=\frac{e^{-\lambda_1}\frac{\lambda_1^{n_1}}{n_1!}\cdot ... \cdot e^{-\lambda_k}\frac{\lambda_k^{n_k}}{n_k!}}{e^{-(\lambda_1+...\lambda_k)}\frac{(\lambda_1+...+\lambda_k)^n}{n!}}=...$$



$$...=\frac{\prod e^{-\lambda_i}\frac{\lambda_i^{n_i}}{n_i!}}{e^{\sum\lambda_i}\frac{(\lambda_1+...\lambda_k)^n}{n!}}=\binom{n}{n_1,...,n_k}\frac{\prod e^{-\lambda_i}\lambda_i^{n_i}}{e^{-\sum\lambda_i}(\lambda_1+...+\lambda_k)^n}=...$$


$$...=\binom{n}{n_1,...,n_k}\frac{(e^{-\lambda_1}\cdot ... \cdot e^{-\lambda_k})}{e^{-\sum\lambda_i}}\frac{\prod \lambda_i^{n_i}}{(\lambda_1+...+\lambda_k)^n}=\binom{n}{n_1,...,n_k}\frac{e^{-\sum\lambda_i}}{e^{-\sum\lambda_i}}\frac{\prod \lambda_i^{n_i}}{(\lambda_1+...+\lambda_k)^n}=...$$

$$ ...=\binom{n}{n_1,...,n_k}\frac{\prod \lambda_i^{n_i}}{(\lambda_1+...+\lambda_k)^{n_1}\cdot ...\cdot(\lambda_1+...+\lambda_k)^{n_k}}=\binom{n}{n_1,...,n_k}\prod(\frac{\lambda_i}{\sum\lambda_i})^{n_i}$$

Osserviamo che : 

$$\frac{\lambda_i}{\sum\lambda_i}\in[0,1]$$

Questa osservazione insieme al coefficiente multinomiale ci riconducono alla distribuzione multinomiale , quindi:

$$\mathbb{P}(Z_1=n_1,...,Z_k=n_k|Z_1+...+Z_k=n)=\binom{n}{n_1,...,n_k}q_1^{n_1}\cdot q_2^{n_2}\cdot ...\cdot q_k^{n_k}$$

$3)$ Così rappresentata questa distribuzione è facilmente riconducibile allo schema della distribuzione casuale di $n$ palline in $k$ scatole , infatti abbiamo esattamente $k$ fattori , che rappresentano le $k$ scatole  con la probabilità $q_i$  che ha una pallina di cascarci dentro . Invece l'esponente $n_i$ rappresenta proprio in numero di palline che si trovano nell' i-esima scatola .
